lp-collaborative-workspace/lp-cw-bridge/lp-cw-bridge-implementation/src/main/java/eu/learnpad/cw/CWXwikiBridge.java

```diff
@@ -362,7 +362,7 @@ public Recommendations getRecommendations(String modelSetId, String artifactId,
 
         for (String userId : potentialUsers) {
             UserData user = new UserData();
-            user.id = userId;
+            user.id = this.removePrefixes(userId);
```

Comment:
Note that it also means that the page `SimulationEnvironment` in the CW must then do the same operation in order to send user ID without that prefix to SIM.